.github/workflows/build.yml

```diff
@@ -19,25 +19,13 @@ jobs:
   Sanity_Checks:
     runs-on: ubuntu-22.04
     steps:
-    # workaround for broken clang on ubuntu runner until fixed: https://github.com/actions/runner-images/issues/8659
-    - uses: claywar/workaround8649@4892524a133793c85e25513fae79fc968e63877c
-      with:
-        os: ubuntu-22.04
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
     - name: Install Dependencies
       run: |
-        # https://mariadb.com/docs/connect/programming-languages/c/install/#connector-c-install-repo-configure-cs
-        # Ubuntu is using an old libmariadb but python3 pip expects it to be newer. Use the mariadb provided deb packages.
-        wget -P /tmp/ https://downloads.mariadb.com/MariaDB/mariadb_repo_setup
-        # Note: there is a sha256sum command here according to the above link, but I am unsure if this file may change with time
-        chmod +x /tmp/mariadb_repo_setup
-        sudo /tmp/mariadb_repo_setup --mariadb-server-version="mariadb-10.6"
-        # Remove the above when it's no longer necessary
-
         sudo apt-get update
-        sudo apt-get install -y software-properties-common cppcheck luajit-5.1-dev luarocks mariadb-server-10.6 mariadb-client-10.6 libmariadb-dev-compat binutils-dev
+        sudo apt-get install -y gcc-13 clang-15 clang-tidy-15 clang-format-15 software-properties-common cppcheck luajit-5.1-dev luarocks mariadb-server-10.6 mariadb-client-10.6 libmariadb-dev-compat binutils-dev
```

Comment:
There's no need to add these to the install section, since these are already present on the build runner machine

Reply:
I seem to be in the habit of adding packages to my Gentoo GitHub workflow where this is required :crying_cat_face: